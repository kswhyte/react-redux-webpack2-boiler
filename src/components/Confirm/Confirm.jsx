import React from 'react';

import './confirm.css'

const Confirm = () => {
    return (
        <div className="confirm-block">
            <h1>Confirm</h1>
        </div>
    );
};

Confirm.propTypes = {};

export default Confirm;
