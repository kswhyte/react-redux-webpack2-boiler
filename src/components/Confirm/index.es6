import Component from './Confirm';
export default Component;
