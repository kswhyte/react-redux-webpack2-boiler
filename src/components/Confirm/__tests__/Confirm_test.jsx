import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import Confirm from '../Confirm';

// import { specs, describe, it } from 'storybook-addon-specifications';

import {storiesOf, describe, it, specs
} from "../../../../.storybook/facade";

import {shallow} from 'enzyme';
import expect from 'expect';


const stories = storiesOf('Confirm', module);

stories.add('Story', () => {
    const confirm = (
      <Confirm/>);

    specs(() => describe('Show a successful alert', () => {
        it('Should render the Confirm component without crashing', () => {
            let output = shallow(confirm);
            expect(output.find('.confirmPatientInfo-wrapper').length).toEqual(1);
        });
    }));
    return confirm;
});
