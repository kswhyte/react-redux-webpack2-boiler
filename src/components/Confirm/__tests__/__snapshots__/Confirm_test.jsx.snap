// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Story 1`] = `
<div
  className="confirm-block"
>
  <h1>
    Confirm
  </h1>
</div>
`;
